modules/ROOT/pages/dataweave-memory-management.adoc

```diff
@@ -36,15 +36,15 @@ This system property determines the size in bytes of the in-memory input and out
 Introduced in Mule 4.2.2, this option controls whether DataWeave uses off-heap memory (the default) or heap memory. DataWeave uses off-heap memory for internal buffering. However, this setting can cause problems on machines that have only a small amount of memory.
 
 * `com.mulesoft.dw.memory_pool_size` +
-Since Mule 4.3.0 DataWeave buffers use off-heap memory from a pool up to a defined size and the rest is allocated using heap memory. This property determines the number of slots in the pool.
+Since Mule 4.3.0, DataWeave buffers use off-heap memory from a pool up to a defined size and allocates the rest using heap memory. This property determines the number of slots in the pool.
 The default number of slots is 60.
 
 * `com.mulesoft.dw.max_memory_allocation` +
-Introduced in Mule 4.3.0, this option determines the size in bytes of each slot in the pool of off-heap memory. The default slot size is 1572864 bytes (1.5 MB).
+Introduced in Mule 4.3.0, this property determines the size in bytes of each slot in the pool of off-heap memory. The default slot size is 1572864 bytes (1.5 MB).
 
 * `com.mulesoft.dw.buffer.memory.monitoring` (experimental) +
-Introduced in Mule 4.3.0, when this is enabled each time a slot from the memory pool is taken or released a message is logged.
-Note that it may be removed or change its behaviour in future versions.
+Introduced in Mule 4.3.0, when this property is enabled each time, a slot from the memory pool is taken or released and a message is logged.
```

Comment:
the `and` is wrong, we can rephrase it to "a message is logged whenever a slot from the memory pool is taken or released"

Reply:
the `,` should go before `each time`